app/code/Magento/Ups/Model/Carrier.php

```diff
@@ -1200,10 +1200,10 @@ function () use ($httpResponse, $debugData) {
                             $debugData['result'] = ['error' => $e->getMessage(), 'code' => $e->getCode()];
                             $this->_logger->critical($e);
                         }
-                        if ($responseResult) {
-                            $jsonResponse = $responseResult->getStatusCode() >= 400 ? '' : $responseResult->getBody();
+                        if ($responseResult && ($responseResult->getStatusCode() < 400)) {
```

Comment:
You do not need additional parenthesis.
```suggestion
                        if ($responseResult && $responseResult->getStatusCode() < 400) {
```